rfcs/xxxx-authorization/text.md

```diff
@@ -0,0 +1,196 @@
+# Authorization
+
```

Comment:
General comment since _expire_ and _expiration_ references are sprinkled though the text: per the call on Dec 8, we clarified that there is no TTL (expiration) requirement for the bearer token. It can be revoked by the owner.

Reply:
I've removed the parts talking about expiration.